src/NJsonSchema.CodeGeneration.CSharp/Models/EnumTemplateModel.cs

```diff
@@ -51,7 +51,18 @@ public IEnumerable<EnumerationItemModel> Enums
         {
             get
             {
+             
                 var entries = new List<EnumerationItemModel>();
+                
+                Dictionary<string, object> Descriptions = new Dictionary<string, object>();
+                Dictionary<string, object> Titles = new Dictionary<string, object>();
+
+                if (_schema.ExtensionData.ContainsKey("elaboration"))
+                    Descriptions = (Dictionary<string, object>)_schema.ExtensionData["elaboration"];
```

Comment:
Why "elaboration"? Is this in the JSON Schema standard?

Reply:
as above